testing/infra/k8s/base/stack/apm-server.yaml

```
@@ -1,31 +1,33 @@
-apiVersion: agent.k8s.elastic.co/v1alpha1
-kind: Agent
+apiVersion: apps/v1
+kind: Deployment
 metadata:
   name: apm-server
   namespace: default
 spec:
-  version: 8.8.0-ba3f07b2-SNAPSHOT
-  mode: fleet
-  policyID: eck-apm-server
-  kibanaRef:
-    name: kibana
-  fleetServerRef:
-    name: fleet-server
-  deployment:
-    replicas: 1
-    strategy:
-      # Can be changed to "Replace" for extremely memory limited environments.
-      type: "RollingUpdate"
-    podTemplate:
-      spec:
-        containers:
-        - name: agent
+  replicas: 1
+  selector:
+    matchLabels:
+      app: apm-server
+  template:
+    metadata:
+      labels:
+        app: apm-server
+    spec:
+      containers:
+        - image: apm-server
```

```suggestion
        - image: docker.elastic.co/apm/apm-server:8.8.0-ba3f07b2-SNAPSHOT
```

Can we please use the published image here, and switch it to "apm-server" in the overlay? The idea is that people can use the base directly, without Tilt.

---

Oh I see! The code is more consistent with the other yaml now :+1: